src/__tests__/utils/fixtures.ts

```diff
@@ -142,6 +142,7 @@ const getTestData = () => {
       is_improvement: true,
       is_regression: false,
       is_meaningful: true,
+      parent_signature: '4769486',
```

Comment:
Tell me if I'm wrong, but my guess is that `parent_signature` is returned only for subtests.
Therefore I'm not sure we should have them by default, rather we should have it only for subtests (so when you'll add tests, you'll add them recursively using `map` when defining the mock).

Reply:
see the discussion with the type

Reply:
We need access to the parent_signature value so we can make a request to get the subtests results, otherwise we can't know which subtests compare results belong to a certain regular compare result (if that makes sense).